src/Entity/Diff/PropertyDiffer.php

```diff
@@ -79,9 +79,9 @@ private function diffPropertyArrays( array $from, array $to ) {
 	private function toDiffArray( Property $property ) {
```

Comment:
Indeed, a `Fingerprint` is sufficient  here. I already thought about creating a separate `FingerprintDiffer` but I think  these 5 lines of code duplication are not worth a new class.